src/test/app/SetJSHook_test.cpp

```diff
@@ -9091,6 +9091,125 @@ class SetJSHook_test : public beast::unit_test::suite
         env(pay(bob, alice, XRP(1)), M("test util_verify"), fee(XRP(1)));
     }
 
+    void
+    test_sto_emplace(FeatureBitset features)
+    {
+        testcase("Test sto_emplace");
+        using namespace jtx;
+
+        Env env{*this, features};
+
+        auto const bob = Account{"bob"};
+        auto const alice = Account{"alice"};
+        env.fund(XRP(10000), alice);
+        env.fund(XRP(10000), bob);
+
+        TestHook hook = jswasm[R"[test.hook](
+            var ASSERT = (x) => {
+              if (!x) {
+                rollback(x.toString(), 0)
+              }
+            }
+            var sto = [
+              17, 0, 97, 34, 0, 0, 0, 0, 36, 4, 31, 148, 217, 37, 4, 94, 132, 183, 45, 0, 0,
+              0, 0, 85, 19, 64, 179, 37, 134, 49, 150, 181, 111, 65, 245, 137, 235, 125, 47,
+              217, 76, 13, 125, 184, 14, 75, 44, 103, 167, 120, 42, 214, 194, 176, 119, 80,
+              98, 64, 0, 0, 0, 0, 164, 121, 148, 129, 20, 55, 223, 68, 7, 231, 170, 7, 241,
+              213, 201, 145, 242, 211, 111, 158, 184, 199, 52, 175, 108,
+            ]
+            var ins = [
+              86, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17,
+              17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17,
+            ]
+            var ans = [
+              17, 0, 97, 34, 0, 0, 0, 0, 36, 4, 31, 148, 217, 37, 4, 94, 132, 183, 45, 0, 0,
+              0, 0, 85, 19, 64, 179, 37, 134, 49, 150, 181, 111, 65, 245, 137, 235, 125, 47,
+              217, 76, 13, 125, 184, 14, 75, 44, 103, 167, 120, 42, 214, 194, 176, 119, 80,
+              86, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17,
+              17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 98, 64, 0, 0, 0, 0,
+              164, 121, 148, 129, 20, 55, 223, 68, 7, 231, 170, 7, 241, 213, 201, 145, 242,
+              211, 111, 158, 184, 199, 52, 175, 108,
+            ]
+            var ans2 = [
+              17, 0, 97, 34, 0, 0, 0, 0, 36, 4, 31, 148, 217, 37, 4, 94, 132, 183, 45, 0, 0,
+              0, 0, 84, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17,
+              17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 17, 85, 19, 64, 179,
+              37, 134, 49, 150, 181, 111, 65, 245, 137, 235, 125, 47, 217, 76, 13, 125, 184,
+              14, 75, 44, 103, 167, 120, 42, 214, 194, 176, 119, 80, 98, 64, 0, 0, 0, 0,
+              164, 121, 148, 129, 20, 55, 223, 68, 7, 231, 170, 7, 241, 213, 201, 145, 242,
+              211, 111, 158, 184, 199, 52, 175, 108,
+            ]
```

Comment:
It would be good if those were written in hexadecimal to match the other test cases.

Reply:
That's a good point

Reply:
See: https://github.com/Xahau/xahaud/pull/448#discussion_r1972952855